src/i18n/locales/en/translations.json

```diff
@@ -544,35 +544,35 @@
     "availability": "Available on other parachains",
-    "availability.tooltip_one": "This asset is also registered on {{count}} another parachain.",
+    "availability.tooltip_one": "This asset is also registered on one other parachain.",
     "availability.tooltip_other": "This asset is also registered on {{count}} other parachains.",
     "isolatedPool": "Isolated Pool created",
     "volume": "Daily volume on Hydration"
   },
-  "wallet.addToken.rugCheck.symbol": "Adjustment of symbol could mean a foul play from developers blah bla blah some more explanation on what the problem could be for people that do not understand those aspects",
-  "wallet.addToken.rugCheck.decimals": "Adjustment in decimal spaces could mean a foul play from developers blah bla blah some more explanation on what the problem could be for people that do not understand those aspects",
-  "wallet.addToken.rugCheck.supply": "{{ name }} token is no longer backed as the admin of the token tampered with the supply you can no longer withdraw all the funds.",
-  "wallet.addToken.toast.register.onLoading": "Registering {{name}} on Hydration as external asset",
-  "wallet.addToken.toast.register.onSuccess": "You registered {{name}} as external asset",
-  "wallet.addToken.toast.add.onSuccess": "You added info about {{name}} to your UI",
-  "wallet.addToken.tooltip.warning": "This asset is registered as an external token. Always conduct your own research before trading.",
-  "wallet.addToken.tooltip.danger": "The creator of this asset has not revoked their admin rights. This means the supply can be minted, burned or frozen at any point. Proceed at your own risk.",
-  "wallet.assets.farmingPositions.empty.desc": "You don’t have any farming position yet, add liquidity and start earning with Hydration.",
+  "wallet.addToken.rugCheck.symbol": "Asset symbols don't usually change. This could be an indication that the asset creator is trying to create the false impression that the asset in question is another asset.",
+  "wallet.addToken.rugCheck.decimals": "Asset decimals don't usually change. This could be an indication that the asset creator is trying to manipulate the supply in order to inflate the value of the asset.",
+  "wallet.addToken.rugCheck.supply": "The creator of the asset {{ name }} has most likely tampered with its total supply. As a result, you may no longer be able to withdraw all your funds to Asset Hub.",
+  "wallet.addToken.toast.register.onLoading": "Registering {{name}} on Hydration...",
+  "wallet.addToken.toast.register.onSuccess": "{{name}} registered on Hydration.",
+  "wallet.addToken.toast.add.onSuccess": "{{name}} added to your UI.",
+  "wallet.addToken.tooltip.warning": "This is an external asset added by yourself. Always conduct your own research before trading.",
+  "wallet.addToken.tooltip.danger": "If the creator of an asset has not revoked their admin rights, they are still able to manipulate the supply. Proceed at your own risk.",
```

Comment:
This is not if... It should warn that they didn't

Reply:
Is this really resolved? I just see closed conversation but no changes to everything I posted.

Reply:
I saw changes from @vgantchev so I assumed it is.

Reply:
No all of the comments were on the changes he made

Reply:
so these tooltips will only display if relevant/applicable, correct?